html/render.js

```diff
@@ -2,9 +2,10 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 
 import HomePagePanel from '../src/index.js'
+import { SceaHomepageSpeciesContainer } from 'atlas-homepage-cards'
 
 const render = (options, target) => {
-  ReactDOM.render(<HomePagePanel {...options} />, document.getElementById(target))
+  ReactDOM.render(<HomePagePanel {...options} CardType={SceaHomepageSpeciesContainer} />, document.getElementById(target))
```

Comment:
Two points:
1. There’s a confusion here between a card type and a container. Why is a type of card assigned a container? See the other comment below where this should go in the other repo anyway.
2. If this prop is hard-coded, why is it a prop at all?